easybuild/easyconfigs/j/JWM/JWM-2.3.5-intel-2016a.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'JWM'
+version = '2.3.5'
+
+homepage = 'https://joewing.net/projects/jwm/'
+description = """JWM is a light-weight window manager for the X11 Window System."""
```

Comment:
window manager? WINDOW MANAGER?! aarggh... I knew this day would come one day :)

first visual review ok, thanks for the contribution!